app/Models/Entry.php

```diff
@@ -79,7 +79,7 @@ public static function fromArray(array $dao): FreshRSS_Entry {
 			$dao['tags'] ?? ''
 		);
 		if (!empty($dao['id'])) {
-			$entry->_id($dao['id']);
+			$entry->_id((string)$dao['id']);
```

Comment:
I think it would be clearer to make the behaviour more clear, like
https://github.com/FreshRSS/FreshRSS/pull/5830/commits/790f3f966e6deef7b1514de7e2c1d2618b963419